1a_single_site/slurm_distributed_workflow/05_validation.Rmd

```diff
@@ -0,0 +1,407 @@
+---
```

Comment:
Looks like this is identical to the version in the parent dir -- any reason not to delete it from here?